src/Components/Header/index.js

```diff
@@ -4,14 +4,17 @@ import ProfileMenu from './ProfileMenu';
 function Header() {
   const token = sessionStorage.getItem('token');
   const logo = `${process.env.PUBLIC_URL}/assets/images/logo.png`;
+  const linkUrl = `http://localhost:3000/auth/home`;
```

Comment:
only the devs are using localhost on production stage. QA team are usin vercel so this would cause them errors. Just use useHistory with the suggested code above.

Reply:
Perfect. I'm going to implement it.